tests/test_cli.py

```diff
@@ -107,3 +108,78 @@ def test_create_user_config():
         "foo": None,
         "main": 3,
     }
+
+
+class MockResponse:  # pylint: disable=too-few-public-methods
+    def __init__(self, data, status_code):
+        self.data = data
+        self.status_code = status_code
+
+    def json(self):
+        return self.data
+
+
+def create_patched_cli():
```

Comment:
Maybe "authenticated_cli"?

Reply:
good point, spend so much on this mocking issue, so haven't even thought about that

Reply:
done @jankatins